devimint/src/main.rs

```diff
@@ -170,14 +170,35 @@ pub async fn latency_tests(dev_fed: DevFed) -> Result<()> {
         anyhow::ensure!(payment_status == tonic_lnd::lnrpc::payment::PaymentStatus::Succeeded);
         ln_receives.push(start_time.elapsed());
     }
+
+    info!("Waiting for a new session");
+    let session_count = cmd!(fed, "dev", "api", "session_count").out_json().await?["value"]
+        .as_u64()
+        .context("session count must be integer")?
+        .to_owned();
+    let start = Instant::now();
+    poll("Waiting for a new session", 6, || async {
+        info!("Awaiting session outcome {session_count}");
+        match cmd!(fed, "dev", "api", "await_session_outcome", session_count)
+            .run()
+            .await
+        {
+            Err(e) => Err(ControlFlow::Continue(e)),
+            Ok(_) => Ok(()),
+        }
+    })
+    .await?;
+    let session_found_in = start.elapsed();
```

Comment:
https://github.com/fedimint/fedimint/pull/4173#discussion_r1477029091 also applies here

> 

I don't think this measures what you expect? If you are somewhere within a session it could take any time between 0s-3m to finish it. If you want to measure session completion time you have to start measuring at the start of a session.